examples/kebechet/README.md

```diff
@@ -26,3 +26,18 @@ The two images below show two sequence diagrams of the behaviour of Kebechet in
 ![Kebechet Schema](../../assets/images/kebechet-schema-internal-trigger-package-release.png)
 
 As you can see, there are different workflows that are triggered to manage your Python dependencies using Thoth's recommendation system. You can request directly for guidance to the service (external trigger) or your repository will be automatically updated with the optimal Python software stack for your project without you having to lift your finger (internal trigger). You can have a detailed description of all the components involved into Kebechet [here](./tooling.md).
+
+
+## Monitoring
+
+- [Tooling](https://github.com/AICoE/data-driven-development/blob/main/examples/kebechet/tooling.md) -> This section describes all components involved and what are used for monitoring and analytics.
+
+- [Metrics](https://github.com/AICoE/data-driven-development/blob/main/examples/kebechet/metrics.md) -> This section show all metrics defined.
```

Comment:
change to `This section describes all the metrics defined for monitoring Kebechet`